internal/plugin/server.go

```diff
@@ -174,7 +174,7 @@ func (plugin *NvidiaDevicePlugin) Serve() error {
 
 			// restart if it has not been too often
 			// i.e. if server has crashed more than 5 times and it didn't last more than one hour each time
-			if restartCount > 5 {
+			if restartCount >= 5 {
 				// quit
 				klog.Fatalf("GRPC server for '%s' has repeatedly crashed recently. Quitting", plugin.rm.Resource())
 			}
```

Comment:
Let's either move this to the start or the end of the loop to make it more consistent. This should also allow us to keep `restartCount > 5` as the conditional.

Reply:
Done